### ISSUE TITLE
Parameterized scenario with simple values and CSV rows

### ISSUE THREAD

**Issue by authorjapps** [authorjapps/zerocode#283]
GIVEN I have defined set of input parameters to drive the tests
WHEN I create a simple value source 
THEN I will run the steps for each value once for each parameter

GIVEN I have defined set of input parameters as CSV to drive the tests
WHEN I create a simple CSV source 
THEN I will run the steps for each row once for each parameter in that row

e.g.

Simple value source
```
    "parameterized": {
        "valueSource":[
            "octocat",
            "userX",
            ...
        ]
    }
```
CSV value source
```
    "parameterized": {
        "csvSource":[
            "octocat, The Octocat, San Francisco, 583231",
            "userX, User X, UK, 847799",
            ...
        ]
    }
```

A full scenario will look like below:
```
{
    "scenarioName": "Fetch and assert GitHub userNames",
    "steps": [
        {
            "name": "get_user_details",
            "url": "/users/${0}",    <--- 1st time:'octocat', 2nd time:'userX'
            "operation": "GET",
            "request": {
            },
            "assertions": {
                "status": 200,
                "body": {
                    "login" : "${0}"  <--- 1st time:'octocat', 2nd time:'userX'
                }
            }
        }
    ],
    "parameterized": {
        "valueSource":[
            "octocat",
            "userX"
        ]
    }
}

```

**Comment by authorjapps** [comment]
link #207

**Comment by authorjapps** [comment]
//TODO- A Wiki page with examples describing how the parameterized tests work

**Comment by authorjapps** [comment]
Available in `version 1.3.10` and higher.

**Comment by authorjapps** [comment]
Help and usage Wiki - https://github.com/authorjapps/zerocode/wiki/Parameterized-Testing-From-CSV-rows
